hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -532,14 +556,20 @@ def setup_occupancy_type(
                     gdf_amenity["Secondary Object Type"] == "apartments", "Secondary Object Type"
                 ] = "residential"
                 gdf = gdf_amenity
+                gdf.drop_duplicates(subset='geometry', inplace=True)
 
             # Remove the objects that do not have a Primary Object Type, that were not
             # overlapping with the land use map, or that had a land use type of 'nan'.
             if "Primary Object Type" in gdf.columns:
                 nr_without_primary_object = len(
                     gdf.loc[gdf["Primary Object Type"].isna()].index
-                ) + len(gdf.loc[gdf["Primary Object Type"] != ""].index)
+                ) + len(gdf.loc[gdf["Primary Object Type"] == ""].index)
```

Comment:
maybe it is better to actually change all the "" cells to None, so from this point onwards you can use only a single filtering with .isna() isntead of having to do both?

Reply:
check